From: =?UTF-8?q?J=C3=A1n=20Tomko?= <jtomko@redhat.com>
Date: Tue, 14 Apr 2015 12:06:44 +0200
Subject: [PATCH] tests: rename testStripIPv6BracketsData to testStripData

For reuse with other Strip* functions.

(cherry picked from commit e892842dfd3c7bad8fbfbfcf0501d01804e9e7c3)
---
 tests/virstringtest.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/virstringtest.c b/tests/virstringtest.c
index a0bfd61..9d0b438 100644
--- a/tests/virstringtest.c
+++ b/tests/virstringtest.c
@@ -522,14 +522,14 @@ testVirStringFreeListCount(const void *opaque ATTRIBUTE_UNUSED)
 }
 
 
-struct testStripIPv6BracketsData {
+struct testStripData {
     const char *string;
     const char *result;
 };
 
 static int testStripIPv6Brackets(const void *args)
 {
-    const struct testStripIPv6BracketsData *data = args;
+    const struct testStripData *data = args;
     int ret = -1;
     char *res = NULL;
 
@@ -766,7 +766,7 @@ mymain(void)
 
 #define TEST_STRIP_IPV6_BRACKETS(str, res)                              \
     do {                                                                \
-        struct testStripIPv6BracketsData stripData = {                  \
+        struct testStripData stripData = {                              \
             .string = str,                                              \
             .result = res,                                              \
         };                                                              \
